### PR TITLE
[SPARK-29803][Python] remove all instances of 'from __future__ import print_function'

### PR DESCRIPTION
**Pull request by tianshizz**
### What changes were proposed in this pull request?

removed all remaining instances of `from __future__ import print_function`

### Why are the changes needed?

deprecate Python 2

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

existing tests

### CODE REVIEW

File: examples/src/main/python/als.py [hunk 1]
```diff
@@ -21,8 +21,6 @@
 
 This example requires numpy (http://www.numpy.org/)
 """
-from __future__ import print_function
-
```
Comment (HyukjinKwon) [inline examples/src/main/python/als.py:25]: Let's don't do this for now ..  we should officially drop Python 2 first.

Comment (tianshizz) [reply examples/src/main/python/als.py:25]: Oh, I see. I guess it's probably more than 6 months away? I can close the two prs if that's the case.

Comment (HyukjinKwon) [reply examples/src/main/python/als.py:25]: Nobody knows but at least that wouldn't happen until Spark 3.0 release which will take a couple of months at least. Let's leave it closed for now.

**Comment by HyukjinKwon** [comment]
Thanks for taking a look, though. I'll close this for now.

**Pull request closed without merging**